install.sh

```diff
@@ -1,4 +1,12 @@
 set -euo pipefail
+VAR_DIR='/var/lib/tensorleap/standalone'
+SCRIPT_LOG=$VAR_DIR/log.log
+
+function log(){
+  echo "[$(date)] [INFO]  $1" >> $SCRIPT_LOG
```

Comment:
does it adds new line after each `echo`?

Reply:
yes